samples/nrf9160/modem_shell/README.rst

```diff
@@ -615,6 +641,28 @@ Examples
 
   Note the syntax for escaping the quotation marks.
 
+----
+
+.. _uart_command:
+
+UART
+====
+
+Disable UARTs for power measurement purposes.
+
+* Disable UARTs for 30 seconds:
+
+  .. code-block:: console
+
+     uart disable 30
+
+* Disable UARTs whenever modem is in sleep and subscribe to modem sleep notifications:
```

Comment:
It could be said that `link msleep --subscribe` is a requirement of the `uart during_sleep disable` command, for example:
```suggestion
* Disable UARTs whenever modem is in sleep (modem sleep notifications must be subscribed to use this feature):
```

Reply:
```suggestion
* Disable UARTs whenever modem is in sleep (modem sleep notifications must be subscribed to in order to use this feature):
```

Reply:
If it's needed, why we don't do it automatically in the SW?

Reply:
That is actually a good question. 😄 Maybe we should just modify the implementation and remove the need for this comment. @hiltunent

Reply:
I had a quick look at this. The modem sleep notification implementation in modem_shell takes an optional parameter for the threshold (in ms). If it is not given by the user, `CONFIG_LTE_LC_MODEM_SLEEP_NOTIFICATIONS_THRESHOLD_MS` is used instead. We shouldn't overwrite a custom value given by the user when using the `uart` command.

It's not possible to query from modem if sleep notifications are enabled or not. But link module could remember the state and we could do a subscribe with the default values from the `uart` command if modem sleep notifications are not yet subscribed.

Reply:
Yep, so I'll remove the subscription from the example.